src/chains/filecoin/filecoin/src/things/block-header.ts

```diff
@@ -171,8 +171,8 @@ class BlockHeader
       timestamp: {
         deserializedName: "timestamp",
         serializedName: "Timestamp",
-        defaultValue: () => {
-          return new Date().getTime() / 1000;
+        defaultValue: literal => {
+          return literal || new Date().getTime() / 1000;
```

Comment:
how's this work? If `literal` is a `0` somehow will this behave appropriately?

Reply:
This should probably be

```suggestion
          return typeof literal !== "undefined" ? literal : (new Date().getTime() / 1000);
```

Reply:
I applied the above change, but FWIW, this field is unused by Ganache